Formula/c/cargo-cache.rb

```diff
@@ -0,0 +1,19 @@
+class CargoCache < Formula
+  desc "Display information on the cargo cache, plus optional cache pruning"
+  homepage "https://github.com/matthiaskrgr/cargo-cache"
+  url "https://github.com/matthiaskrgr/cargo-cache/archive/refs/tags/0.8.3.tar.gz"
+  sha256 "d0f71214d17657a27e26aef6acf491bc9e760432a9bc15f2571338fcc24800e4"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "rust" => :build
+  uses_from_macos "zlib"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    output = shell_output("CARGO_HOME=#{bin} #{bin}/cargo-cache")
```

Comment:
What is the purpose of `CARGO_HOME` here? Is it where to find cargo? Or where to write files?

Reply:
This tool check the cache in your cargo directory which by default is located in `$HOME/.cargo` which isn't guaranteed to actually exist (the program will then gives an error), so I manually override it to a folder that's guaranteed to exist (the test dir). The program luckily will just report "nothing used" with a non-cargo folder instead of erroring out.

Reply:
That's not the test dir, it's the place where the binaries for cargo-cache are stored. This would be better:
```suggestion
    mkdir testpath/".cargo"
    ENV["CARGO_HOME"] = testpath
    output = shell_output("#{bin}/cargo-cache")
```

Reply:
Yeah I mistook the bindir for the test dir, sorry. (Also the `mkdir` is unnecessary because we already set `CARGO_HOME`)